docs/en/sql-reference/functions/date-time-functions.md

```diff
@@ -4300,8 +4300,8 @@ Result:
 
 For a time interval starting at 'StartTime' and continuing for 'Duration' seconds, it returns an array of moments in time, consisting of points from this interval rounded down to the 'Size' in seconds. 'Size' is an optional parameter set to 1800 (30 minutes) by default.
 This is necessary, for example, when searching for pageviews in the corresponding session.
-Accepts DateTime and DateTime64 as 'StartTime' argument. For DateTime, 'Duration’ and ’Size’ arguments must be `UInt32`. For ’DateTime64’ they must be `Decimal64`.
-Returns an array of DateTime/DateTime64 (return type matches the type of ’StartTime’). For DateTime64, the return value's scale can differ from the scale of ’StartTime’ --- the highest scale among all given arguments is taken.
+Accepts DateTime and DateTime64 as 'StartTime' argument. For DateTime, 'Duration' and 'Size' arguments must be `UInt32`. For 'DateTime64' they must be `Decimal64`.
+Returns an array of DateTime/DateTime64 (return type matches the type of 'StartTime'). For DateTime64, the return value's scale can differ from the scale of 'StartTime' --- the highest scale among all given arguments is taken.
```

Comment:
```suggestion
Accepts `DateTime` and `DateTime64` as 'StartTime' argument. For `DateTime`, 'Duration' and 'Size' arguments must be `UInt32`. For `DateTime64`, they must be `Decimal64`.
Returns an array of `DateTime` or `DateTime64` (return type matches the type of 'StartTime'). For `DateTime64`, the return value's scale can differ from the scale of 'StartTime' --- the highest scale among all given arguments is taken.
```

Reply:
The point of this PR is to get rid of non-standard quotation marks, see my changes in `utils/check-style/check-style`.